README.md

```diff
@@ -1,13 +1,51 @@
-# KituraContracts
+<p align="center">
+<a href="http://kitura.io/">
+<img src="https://raw.githubusercontent.com/IBM-Swift/Kitura/master/Sources/Kitura/resources/kitura-bird.svg?sanitize=true" height="100" alt="Kitura">
+</a>
+</p>
+
```

Comment:
At the bottom of this file we should have another section
## License
This library is licensed under Apache 2.0. Full license text is available in LICENSE (this last word should be a fully qualified link to our LICENSE file within this repository).